frontend/sac-mobile/components/input.tsx

```diff
@@ -0,0 +1,34 @@
+import { StyleSheet, Text, TextInput, View } from 'react-native';
+
+type InputProps = {
```

Comment:
```suggestion
interface InputProps {
```

Reply:
and instead of taking all these props, you can just allow the InputProps to extend TextInputProps 

`interface InputProps extends TextInputProps`